pom.xml

```diff
@@ -129,10 +130,10 @@
         <version.maven.plugin.jar>2.6</version.maven.plugin.jar>
         <version.maven.plugin.dependency>2.8</version.maven.plugin.dependency>
 
-        <spring-boot.version>1.3.2.RELEASE</spring-boot.version>
+        <spring-boot.version>2.1.6.RELEASE</spring-boot.version>
```

Comment:
I think the right thing would be to remove this dependency as this has been moved to https://github.com/ff4j/ff4j-spring-boot-starter-parent

Reply:
yeah totally, other updrates are related to thymeleaf v3 and make sens, will probable merge and then remove the spring-boot dependency yeah.

Reply:
+ Spring boot 2.x expect java 11 where the ff4j v1 should stay compliant with jdk8.